docs/useful_tools.md

```diff
@@ -227,31 +227,34 @@ You should obtain a respose similar to:
 ```json
 [
   {
-    "dog": [
-      402.9117736816406,
-      124.19664001464844,
-      571.7910766601562,
-      292.6463623046875
+    "class_name": "dog",
```

Comment:
Maybe you could add some description for the added test tool in the `Test deployment` part as well.
### 4. Test deployment